scripts/gen-corunner.py

```diff
@@ -1,69 +1,109 @@
 #! /usr/bin/env python3
 #
-# This script generates the code of FLASH co-runners, as a finite sequence
-# of unconditional branches, that eventually loops:
+# This script generates the code of two types of co-runners:
+# * FLASH co-runners, as a finite sequence of unconditional branches, that eventually loops:
```

Comment:
It was not clear to me this was a bullet list, as its spans over many lines, and formatting gets in the way.
Maybe just write something like

```
1) FLASH co-runners

...

2) SRAM co-runners
```